pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/cpd/testdata/sample-plsql.txt

```diff
@@ -5,45 +5,45 @@ L1
     [REPLACE]                               11        18
 L2
     [PACKAGE]                               1         8
-    ["test_schema"]                         9         22
+    ["TEST_SCHEMA"]                         9         22
```

Comment:
hm... isn't that an identifier? ~If so, then we probably shouldn't uppercase it...~

Ok, it is an identifier, a "quoted literal"... In that sense, it is not different from e.g. a variable name, and we should indeed uppercase it...

Just double checking, can you confirm?

Reply:
From the plsql [doc](https://docs.oracle.com/en/database/oracle/oracle-database/21/lnpls/plsql-language-fundamentals.html#GUID-C4B3F788-770E-48F8-9A44-ACD7977B1545):
> A quoted user-defined identifier is case-sensitive, with one exception: If a quoted user-defined identifier, without its enclosing double quotation marks, is a valid ordinary user-defined identifier, then the double quotation marks are optional in references to the identifier, and if you omit them, then the identifier is case-insensitive. 

Maybe we should only uppercase quoted identifiers that fit this criteria. E.g. `"not id"` will not be uppercased, but `"an_id"` is uppercased, because it can be referenced as `An_Id` for instance (without quotes), which would itself be uppercased.